west.yml

```diff
@@ -126,6 +136,20 @@ manifest:
       path: modules/alexa-embedded
       revision: face92d8c62184832793f518bb1f19379538c5c1
       remote: alexa
+    # TODO: Add below remotes as submodules to the connectedhomeip
```

Comment:
Perhaps we should wait for the west 0.9.0 with the submodule feature given it's going to be release soon? @carlescufi @mbolivar-nordic

Reply:
@mbolivar-nordic  do you know when west 0.9 version might be available to use?

Reply:
west.yml was updated, as west 0.9.0 is available